### ISSUE TITLE
error in execution

### ISSUE THREAD

**Issue by siva90144** [VinayKumarBM/playwright-cucumber-sample#1]
When trying to execute its giving error as below. is something i am missing?
npm run test:tags @validSearch

> npx cucumber-js -p runner --tags

error: option '-t, --tags <EXPRESSION>' argument missing
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! playwright-cucumber-sample-project@1.0.0 test:tags: `npx cucumber-js -p runner --tags`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the playwright-cucumber-sample-project@1.0.0 test:tags script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

**Comment by VinayKumarBM** [comment]
I will try to have a look at it over the weekend

**Comment by VinayKumarBM** [comment]
@siva90144 : I think you are not running this on cmd prompt. Please try running in command prompt and let me know.
If you want to run on shell command make these changes 
`npm run test:tags "@validSearch"`

Please look into this [latest run](https://github.com/VinayKumarBM/playwright-cucumber-sample/actions/runs/3723600680) where it is working with same command.

**Comment by siva90144** [comment]
@VinayKumarBM  thanks for response. I executed from VS code power shell and command line. for both, it's not picking any scenarios. So, I tried npm run test. still the same. But @validSearch tag is there
PFA...
![image](https://user-images.githubusercontent.com/19613704/209723569-140aeec6-014a-46ba-8fb7-2319b66f6dc9.png)
![image](https://user-images.githubusercontent.com/19613704/209723669-3f999670-152c-48f9-93a3-a42175bacad2.png)
